tests/testthat/test-error-handling.R

```diff
@@ -0,0 +1,59 @@
+context("error handling test")
+test_that("Remove error handling test", {
```

Comment:
Should we have one explicitly for 'stop'?

Reply:
I'll add the test